wdfn-server/waterdata/tests/test_views.py

```diff
@@ -221,35 +210,32 @@ def setUp(self):
         self.network = 'monitoring-locations'
         self.format = 'json'
 
-    @mock.patch('waterdata.services.ogc.get_networks')
+    @mock.patch('waterdata.services.ogc.MonitoringLocationNetworkService.get_networks')
     def test_networks(self, network_mock):
-        network_mock.return_value.status_code = 200
+        network_mock.return_value = json.loads(MOCK_NETWORKS_RESPONSE)
         response = self.app_client.get('/networks/')
         # Assert network listing works
         assert response.status_code == 200
 
-    @mock.patch('waterdata.services.ogc.get_networks')
+    @mock.patch('waterdata.services.ogc.MonitoringLocationNetworkService.get_networks')
     def test_one_network(self, network_mock):
-        network_mock.return_value.status_code = 200
+        network_mock.return_value = json.loads(MOCK_NETWORK_RESPONSE)
         response = self.app_client.get('/networks/{}/'.format(self.network))
         # Assert a known network works in gui
         assert response.status_code == 200
 
-    @mock.patch('waterdata.services.ogc.get_networks')
+    @mock.patch('waterdata.services.ogc.MonitoringLocationNetworkService.get_networks')
     def test_invalid_network(self, network_mock):
-        network_mock.return_value.status_code = 200
-        network = 'monitoring-locations-invalid'
-        url = '/networks/{}/'.format(network)
-        response = self.app_client.get(url)
+        network_mock.return_value = {}
+        response = self.app_client.get('/networks/')
         # Assert an invalid network in gui does not 404
```

Comment:
Is this comment still correct? Seems like the opposite is happening.

Reply:
Thanks. I meant to delete this test.